modules/ROOT/pages/quotas-and-limits.adoc

```diff
@@ -31,13 +31,14 @@ Prompts in other languages are not fully supported.
 * Unique prompts
 +
 Duplicate prompt names or instructions are not allowed.
-* 20000 characters for Einstein prompts. 
+* 20,000 characters for Einstein prompts. 
```

Comment:
duplicated "instructions" is allowed for Generic Action with Einstein (the new experience). It's still not allowed in the legacy experience (because Textract doesn't allow it)